Makefile

```diff
@@ -5,6 +5,7 @@ mandir ?= $(PREFIX)/share/man
 DOCKER ?= podman
 DOCS_VERSION ?= next
 RUN_TESTS ?= ci
+BACKEND ?= osbuild-composer
```

Comment:
I think this should default to lorax-composer, the code isn't dead yet, and this is what it should be tested against by default.

I think it should run the tests twice, once against the default, and again against osbuild-composer.
And osbuild-composer's gating tests should also run this with the osbuild-composer backend set.

Reply:
Yeah, agreed.